pkg/providers/aws/provider_postgres.go

```diff
@@ -214,6 +218,79 @@ func (p *AWSPostgresProvider) createRDSInstance(ctx context.Context, cr *v1alpha
 	}}, croType.StatusMessage(fmt.Sprintf("creation successful, aws rds status is %s", *foundInstance.DBInstanceStatus)), nil
 }
 
+// Tags RDS resources
+func (p *AWSPostgresProvider) TagRDSPostgres(ctx context.Context, cr *v1alpha1.Postgres, rdsSvc rdsiface.RDSAPI, foundInstance *rds.DBInstance) (croType.StatusMessage, error) {
+	logrus.Infof("Adding Tags to RDS instance %s", *foundInstance.DBInstanceIdentifier)
+	// get the environment from the CR
+	// set the tag values that will always be added
+	defaultOrganizationTag := resources.GetOrganizationTag()
+
+	//get Cluster Id
+	clusterId, _ := resources.GetClusterId(ctx, p.Client)
+	// Set the Tag values
+
+	rdsTag := []*rds.Tag{
+		{
+			Key:   aws.String(defaultOrganizationTag + "clusterId"),
+			Value: aws.String(clusterId),
+		},
+		{
+			Key:   aws.String(defaultOrganizationTag + "resource-type"),
+			Value: aws.String(cr.Spec.Type),
+		},
+		{
+			Key:   aws.String(defaultOrganizationTag + "resource-name"),
+			Value: aws.String(cr.Name),
+		},
+	}
+	if cr.ObjectMeta.Labels["productName"] != "" {
+		productTag := &rds.Tag{
+			Key:   aws.String(defaultOrganizationTag + "product-name"),
+			Value: aws.String(cr.ObjectMeta.Labels["productName"]),
+		}
+		rdsTag = append(rdsTag, productTag)
+	}
+
+	inputRdsInstance := &rds.AddTagsToResourceInput{
```

Comment:
I don't think we need to declare this as a variable, and can just pass it directly into `AddTagsToResource`

I know it makes it less readable, but it does sorten the code. We have a lot of logic to add labels. 

What do you think?

Reply:
I like readability but yea the var is only used once, so will remove it